src/server_manager/scripts/get_version.mjs

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import url from "url";
+import path from "path";
+import {readFile} from "fs/promises";
+
+const __filename = url.fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+export async function get_version() {
+  const {version} = JSON.parse(await readFile(path.resolve(__dirname, "../package.json")));
```

Comment:
I wonder if something like `path.resolve(require.resolve("outline-manager"), "package.json")` makes sense or works, so that we can place this script anywhere without having to update the `..` when moving it.

Reply:
Mm, right, I just said something similar on @jyyi1's pr 😅 - do as i say not as i do i guess

Reply:
er, i think this isn't easily doable - we'd have to set up module aliasing at the root level, which can be done a couple ways

either, with a package like [link-module-alias](https://github.com/Rush/link-module-alias), which seems a bit sketchy, or

by using [subpath import maps](https://nodejs.org/api/packages.html#subpath-imports) which is an ESM feature and would require use of the experimental `import.meta.resolve` method (we'd have to remember to always run node with certain flags. This gets easier if we add action support to mjs files, we could just set the node flags there)

we could _also_ add a pre-compile step but that seems - I'd like to avoid that

if you feel it's worth it lmk, otherwise i'll merge this EOD

Reply:
No need to change